webroot/js/local.js

```diff
@@ -12,6 +12,14 @@ jQuery(document).on('ready', function() {
     });
   }
 
+  jQuery('[role=datetime-picker]').each(function() {
```

Comment:
:+1: to moving js code in local.js but this file is gonna unmanageable soon if just keep dumping code in here without proper organization.